framework/tlvf/yaml/tlvf/WSC/WSC_Attributes.yaml

```diff
@@ -429,3 +429,254 @@ cWscVendorExtWfa:
   vs_data:
     _type: uint8_t
     _length: []
+
+cWscAttrVersion:
+  _type: class
+  _is_tlv_class: False
```

Comment:
Why is this not a TLV class? It has a length field which (in case of the dynamic ones) should be updated based on the length of the data.

It is BTW not needed to set this explicitly because it defaults to False anyway. But good that you did it, otherwise I wouldn't have noticed :-)